### PR TITLE
CONCF-407 set WikiaTopsAds width to 1064 so it fits leaderboard and topbutton

### PR DESCRIPTION
**Pull request by bognix**
ping @vforge @Warkot [@gabrys | @nandy-andy]

**DON'T MERGE BEFORE 'OK' FROM ADENG TEAM**

### CODE REVIEW

**Comment by vforge** [comment]
LGTM

**Comment by Warkot** [comment]
LGTM :+1:

File: extensions/wikia/AdEngine/js/AdLogicPageDimensions.js [hunk 1]
```diff
@@ -77,6 +77,7 @@ define('ext.wikia.adEngine.adLogicPageDimensions', [
 			mediaQueriesToCheck.twoColumns = 'screen and (min-width: 1064px)';
```
Comment (hakubo) [inline extensions/wikia/AdEngine/js/AdLogicPageDimensions.js]: how about using 
```js
$.extend(mediaQueriesToCheck, {
...
});
```

Comment (bognix) [inline extensions/wikia/AdEngine/js/AdLogicPageDimensions.js]: jQuery is not included here. What is more I remember there was an experiment for running Ads in Head and as far as I understand adding jQuery as a dependency wouldn't be optimal solution

**Pull request merged**